### PR TITLE
FRW-19 Added page about support Symfony 6

### PR DESCRIPTION
**Pull request by geega**
Ticket: https://spryker.atlassian.net/browse/FRW-19

### CODE REVIEW

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+originalLink: https://documentation.spryker.com/2021080/docs/symfony-6-integration
+originalArticleId: d5e96c3b-3ed6-49ed-982c-aa641e09b559
+redirect_from:
+  - /2021080/docs/symfony-6-integration
+  - /2021080/docs/en/symfony-6-integration
+  - /docs/symfony-6-integration
+  - /docs/en/symfony-6-integration
+  - /v6/docs/symfony-6-integration
+  - /v6/docs/en/symfony-6-integration
+  - /docs/scos/dev/technical-enhancements/symfony-6-integration.html
```
Comment (asmarovydlo) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:15]: Note for the TW, please pay attention to those redirects, they should be removed I guess. CC @AlexSlawinski

**Review by asmarovydlo — DISMISSED** [review]

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md (hunk 1 above)
Comment (AlexSlawinski) [reply docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:15]: I can remove them, sure. Is that what you want @asmarovydlo ?

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+---
+
+Spryker supports Symfony 6 that was released in December 2022. We tried to keep BC for all three major versions of Symfony, but due to some changes in version 5, we had to partially drop support for Symfony 4 in October 2022.
```
Comment (AlexSlawinski) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:18]: What does BC stand for here @asmarovydlo?

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Even though Spryker still supports older versions of Symfony, it doesn’t mean that you can install them in your project. This is mainly because other packages you use or Spryker uses have different requirements. Anyways, you should always try to keep your dependencies updated.
+
+{% endinfo_block %}
+
+<a name="changes"></a>
+
+## Main changes in Symfony 6
+
+Symfony 6 has a new cycle of innovations that starts one that will also last two years, on a modernized codebase that has been cleaned up from the dead weight of the past.
+The big news of Symfony 6 is that PHP 8.0 is now the minimum required version.
```
Comment (AlexSlawinski) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:31]:
```suggestion
The major change included in Symfony 6 is PHP 8.0 being the minimum required version of PHP.
```

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+The code of Symfony 6 has been updated. You can take advantage of all the new features in PHP.
+For example, the code includes PHP 8 [attributes](https://www.php.net/manual/fr/language.attributes.overview.php), more expressive and rigorous type declarations, etc.
```
Comment (AlexSlawinski) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:33]:
```suggestion
For example, the code includes PHP 8 [attributes](https://www.php.net/manual/fr/language.attributes.overview.php), more expressive and rigorous type declarations, and more.
```

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+Read more [CHANGELOG-6.0](https://github.com/symfony/symfony/blob/6.0/CHANGELOG-6.0.md).
```
Comment (AlexSlawinski) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:35]:
```suggestion
Read more at [CHANGELOG-6.0](https://github.com/symfony/symfony/blob/6.0/CHANGELOG-6.0.md). 
```

**Review by AlexSlawinski — CHANGES_REQUESTED** [review]

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md (hunk 2 above)
Comment (asmarovydlo) [reply docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:18]: The support of Symfony 4 has been already dropped, so it's ok.

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+Even though Spryker still supports older versions of Symfony, it doesn’t mean that you can install them in your project. This is mainly because other packages you use or Spryker uses have different requirements. Anyways, you should always try to keep your dependencies updated.
+
+{% endinfo_block %}
+
+<a name="changes"></a>
+
+## Main changes in Symfony 6
+
+Symfony 6 has a new cycle of innovations that starts one that will also last two years, on a modernized codebase that has been cleaned up from the dead weight of the past.
```
[the remark this answers is not in the document]
Comment (asmarovydlo) [reply docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:30]: Yes, a `cycle` @AlexSlawinski

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md (hunk 1 above)
Comment (asmarovydlo) [reply docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:15]: Yes, that is what I think we should do. But please confirm with someone from TW team.

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md (hunk 2 above)
Comment (AlexSlawinski) [reply docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:18]: > 

Unsure what that means Anton - I was just asking what does BC mean / what does it stand for?

Comment (asmarovydlo) [reply docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:18]: Ah, sorry
> I was just asking what does BC mean


BC - backward compatibility.

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+Although Spryker still supports older versions of Symfony,you should avoid installing them in your project. This is because any other packages you or Spryker use have different requirements. Always try to keep your dependencies updated.
+
+{% endinfo_block %}
+
+<a name="changes"></a>
+
+## Main changes in Symfony 6
+
+Symfony 6 has a new cycle of innovations that starts one that will also last two years, on a modernized codebase that has been cleaned up from the dead weight of the past.
```
Comment (AlexSlawinski) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:22]:
```suggestion
Symfony 6 has a new cycle of innovations. When it starts, one cycle lasts two years, on a modernized codebase that has been cleaned up from the dead weight of the past.
```

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
+---
+title: Integrating Symfony 6
+description: Learn about the main changes in the new Symfony version 6
+last_updated: Dec 7, 2022
+template: howto-guide-template
+originalLink: https://documentation.spryker.com/2021080/docs/symfony-6-integration
+originalArticleId: d5e96c3b-3ed6-49ed-982c-aa641e09b559
+---
+
+Spryker supports Symfony 6 that was released in December 2022. We tried to keep BC for all three major versions of Symfony, but due to some changes in version 5, we had to partially drop support for Symfony 4 in October 2022.
```
Comment (AlexSlawinski) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md:10]:
```suggestion
Spryker primarily supports Symfony 6 that was released in December 2022. Backwards compatibility remains for all three major Symfony versions, but support for Symfony 4 was partially dropped in October 2022.
```

**Review by AlexSlawinski — APPROVED** [review]

**Pull request merged**